nexus/server/src/main.rs

```diff
@@ -209,50 +208,36 @@ impl NexusBackend {
         }
     }
 
-    async fn validate_peer<'a>(&self, peer_type: i32, peer: &Peer) -> anyhow::Result<()> {
-        if peer_type != 6 {
-            let peer_executor = self.get_peer_executor(peer).await.map_err(|err| {
+    async fn validate_peer<'a>(&self, peer: &Peer) -> anyhow::Result<()> {
+        //if flow handler does not exist, skip validation
+        if self.flow_handler.is_none() {
+            return Ok(());
```

Comment:
Should this always happen? A lot of our peer types are not usable from the query layer and having flow not validate them doesn't seem strong enough.